website/docs/docs/cloud/billing.md

```diff
@@ -94,6 +94,90 @@ There are 2 options to disable models from being built and charged:
+
+Many dbt Cloud users are utilizing views, which don’t always need to be rebuilt every time you run a job. For any jobs that contain views that _do not_ include macros that dynamically generate code (for example, case statements) based on upstream tables and _do not_ have tests, you can implement the following steps:
+
+1. Go to your current production deployment job in dbt Cloud.
+2. Modify your command to include: `-exclude config.materialized:view`.
+3. Save your job changes.
+
+If you have views that contain macros with case statements based on upstream tables, these will need to be run each time to account for new values. If you still need to test your views with each run, please use the documentation in the next section to create a custom selector. 
+
+#### Excluding views while still running tests
+
+Running tests for views in every job run can help keep data quality intact and save you from the need to rerun failed jobs. To exclude views from your job run while running tests, you can follow these steps to create a custom [selector](https://docs.getdbt.com/reference/node-selection/yaml-selectors) for your job command. 
+
+1. Open your dbt project in the dbt Cloud IDE.
+2. add a file called `selectors.yml` in your top-level project folder.
+3. In the file, put the following code:
+
+   ```yaml 
+    `selectors:
+      - name: skip_views_but_test_views
+        description: >
+          A default selector that will exclude materializing views
+          without skipping tests on views.
+        default: true
+        definition:
+          union:
+            - union: 
+              - method: path
+                value: "*"
+              - exclude: 
+                - method: config.materialized
+                  value: view
+            - method: resource_type
+              value: test`
+
+    ```
+    
+4. Save the file and commit it to your project.
+5. Modify your dbt Cloud jobs to include `--selector skip_views_but_test_views`.
+
+#### Building only changed views
+
+If you would like to ensure that you are building views whenever the logic is changed, create a merge job that gets triggered when code is merged into main: 
+
+1. Ensure you have a Slim CI job setup in your environment.
+2. Create a new [dbt Cloud job](https://docs.getdbt.com/docs/deploy/deploy-jobs#create-and-schedule-jobs) and call it “Merge Job."
+3. Set the [environment](https://docs.getdbt.com/docs/deploy/deploy-environments#types-of-environments) to your CI environment. 
+4. Set Commands to: `dbt run -s state:modified+`.
+    - Note: It's not using dbt build because it already used Slim CI to both run and test the code that just got merged into main, so executing a build command in this context is unnecessary.
+5. Under the execution settings, select the default production job to compare changes against:
+    - Defer to a previous run state — Select the “Merge Job” you created so the job compares and identifies what has changed since the last merge.
```

Comment:
```suggestion
    - **Defer to a previous run state** &mdash; Select the “Merge Job” you created so the job compares and identifies what has changed since the last merge.
```